Meta/lb-cherry-picks.py

```diff
@@ -318,13 +318,11 @@
     "https://github.com/LadybirdBrowser/ladybird/pull/15": "Still used in Serenity",
     "https://github.com/LadybirdBrowser/ladybird/pull/16": "Ladybird-specific",
     "https://github.com/LadybirdBrowser/ladybird/pull/17": "Ladybird-specific",
-
```

Comment:
These empty lines seem to be there on purpose.

Reply:
no real way to keep them without fmt: skip i think